pkg/rbac/authorizationMiddleware.go

```diff
@@ -0,0 +1,25 @@
+package rbac
+
+import (
+	"fmt"
+	"net/http"
+
+	"k8s.io/klog/v2"
+)
+
+func AuthorizeUser(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+
+		//get user cluster-scoped resources and cache:
+		// clientToken := r.Context().Value(ContextAuthTokenKey).(string)
+		// uid := cache.tokenReviews[clientToken].tokenReview.UID
+		// _, err := Instcache.pool.Query(context.Background(), `Select Count(*) from search.resources`, nil)
+		// fmt.Println("Instacache pool works", err)
+		_, err := Instcache.checkUserResources()
+		if err != nil {
+			klog.Warning("Unexpected error while obtaining cluster-scoped resources.", err)
+		}
+		fmt.Println("Finished getting cluster-scoped resources. Now Authorizing..") //place-holder comment.
```

Comment:
Try using `klog.Info()`.  Makes it easier to later adjust with `.V(*)` as needed.